protocol/avdl/keybase1/teams.avdl

```diff
@@ -181,7 +181,7 @@ protocol teams {
     array<UserVersion> writers;
     array<UserVersion> readers;
     array<UserVersion> bots;
-    array<UserVersion> restrictedBots;
+    map<UserVersion, TeamBotSettings> restrictedBots;
```

Comment:
this would warrant a cache version difference unless you've tried it

Reply:
this isn't stored on disk anywhere afaik, just the arg to perform the change unless i'm missing something.

Reply:
oh yeah